python/lsst/obs/base/script/convertGen2RepoToGen3.py

```diff
@@ -156,6 +158,9 @@ def convert(gen2root, gen3root, instrumentClass, calibFilterType,
     calibs : `str`, optional
         Path to the gen2 calibration repository to be converted.
         If a relative path, it is assumed to be relative to ``gen2root``.
+    reruns : `list` [`str`], optional
+        List of reruns to convert. They will be placed in the
+        ``instrumentClass.getName()`` collection.
```

Comment:
Or is it `shared/instrument/converted`?

Reply:
Thanks for catching that.

Reply:
Each rerun should go into a different collection, not the same one, and their names should be derived from the Gen2 repo names somehow.

Reply:
@TallJimbo Is there a way we can codify that with some unittests in obs_base, before rolling it out to ci_hsc? If we want to enforce it, we should do so at the deepest level, probably on its own ticket.

Reply:
I think the problem will become apparent as soon as reruns contain the same output datasets. You can only have one dataset with a given dataId per collection and reruns tend to have many duplicate datasets by definition.

Reply:
If the script generates the chainName and runName passed to the Rerun from the path, unless explicitly overridden, that should cover it pretty well.

Tests in obs_base are of course _possible_, but making a useful mock repo to convert is probably at least a week of work on it's own, and with that ratio I'm willing to live with just coverage in ci_hsc.